test/smoke/src/areas/positron/console/consoleInput.test.ts

```diff
@@ -23,24 +30,35 @@ export function setup(logger: Logger) {
 				const inputCode = `val = input("Enter your name: ")
 print(f'Hello {val}!')`;
 
-				await app.workbench.positronConsole.pasteCodeToConsole(inputCode);
+				await expect(async () => {
+					await app.workbench.positronConsole.pasteCodeToConsole(inputCode);
+
+					await app.workbench.positronConsole.sendEnterKey();
 
-				await app.workbench.positronConsole.sendEnterKey();
+					await app.workbench.positronConsole.waitForConsoleContents((contents) => contents.some((line) => line.includes('Enter your name:')) );
 
-				await app.workbench.positronConsole.waitForConsoleContents((contents) => contents.some((line) => line.includes('Enter your name:')) );
+					// slight wait before starting to type
+					await app.code.wait(200);
 
-				await app.workbench.positronConsole.typeToConsole('John Doe');
+					await app.workbench.positronConsole.typeToConsole('John Doe');
 
-				await app.workbench.positronConsole.sendEnterKey();
+					await app.workbench.positronConsole.sendEnterKey();
 
-				await app.workbench.positronConsole.waitForConsoleContents((contents) => contents.some((line) => line.includes('Hello John Doe!')) );
+					await app.workbench.positronConsole.waitForConsoleContents((contents) => contents.some((line) => line.includes('Hello John Doe!')) );
+				}).toPass();
```

Comment:
Should probably add a timeout.. just in case.

Reply:
Done.